src/test/java/de/tum/in/www1/artemis/exercise/quizexercise/QuizExerciseFactory.java

```diff
@@ -45,9 +62,10 @@ public static QuizExercise createQuizForExam(ExerciseGroup exerciseGroup) {
     }
 
     /**
-     * initializes a quiz with all different types of questions
+     * Initializes a quiz exercise with different types of questions.
+     * After initialization, the quiz consists of one multiple choice, one drag and drop, and one short answer question.
```

Comment:
This is only true if the Quiz did not contain Questions before. The method adds questions but does not remove potentially existing ones. Maybe update the comment or remove existing questions before adding the new ones.